sdk/storage/azure-storage-blobs/BREAKINGCHANGES.md

```diff
@@ -0,0 +1,9 @@
+# Release History
+
+## 1.0.0-beta.3 (2020-10-13)
+
+* Blob uploading/creating is now by default non-overwriting
```

Comment:
Please specify the exact API that's non-overwriting. This line is not entirely correct.